### DISCUSSION TITLE
AI deck streaming speed

### DISCUSSION THREAD

**Discussion by Rwik2000** [bitcraze/discussions#1017] [Q&A]
I am using the wifi-img-streamer example to stream images from the aideck to my computer. I generally get speeds in the range of 3-7 FPS. I see others in the discussion threads also reporting a speed of around 3 FPS. Is there a way to make the streaming faster? Can the streaming be ramped up to atleast 10-15 Hz consistently? 

Is there a way to bypass the communication between stm32 and gap8 i.e. stm32 sends the raw data from the camera directly to the wifi stream?

**Comment by knmcguire** [comment]
Hi! Very sorry for the late reply here. Currently there is not really a way to increase the FPS and we haven't figured out how to fix it properly yet. However, there has been some papers out of the aideck or similar firmware that achieved a higher FPS (not sure about streaming thought) https://ieeexplore.ieee.org/document/9462495

So the answer is, it is probably possible to increase the FPS but we haven't had the time to properly figure this out due to other projects and us being understaffed. I've made an issue though that will remind us to look at this in the future: https://github.com/bitcraze/aideck-gap8-examples/issues/137
